makefile

```diff
@@ -0,0 +1,3 @@
+build-scorecard-action: ## Runs go build on repo
+	# Run go build and generate scorecard executable
+	CGO_ENABLED=0 go build -trimpath -a -tags netgo -ldflags '$(LDFLAGS)'
```

Comment:
`LDFLAGS` is undefined. Maybe add an LDFLAGS var like [here](https://github.com/ossf/scorecard-webapp/blob/main/Makefile#L1).

Also, explicitly call out the binary name this command will output using `-o scorecard-action`